tests/snmp/test_snmp_memory.py

```diff
@@ -25,6 +25,12 @@ def get_parameter(request):
     percent = request.config.getoption("--percentage") or 4
     return percent
 
+def get_percentage_threshold(total_mem):
+    if total_mem > 2 * 1024 * 1024:
```

Comment:
Can you add a comment to explain why we need adjust the threshold for small mem platforms?